data/presets/X-Pressive/X-Pressive1.xpf

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0"?>
+<!DOCTYPE lmms-project>
+<lmms-project version="1.0" creator="LMMS" creatorversion="1.2.0-rc2.6" type="instrumenttracksettings">
+  <head/>
+  <instrumenttracksettings muted="0" type="0" name="X-Pressive" solo="0">
```

Comment:
`name="X-Pressive"`

The preset name that shows up when you load an instrument isn't the name of the `.xpf` but this setting. You need to edit the name in the rename dialogue on the top of the plugin.